RFS/src/main/java/com/rfs/cms/ScopedWorkCoordinatorHelper.java

```diff
@@ -0,0 +1,43 @@
+package com.rfs.cms;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+@Slf4j
+public class ScopedWorkCoordinatorHelper {
```

Comment:
'Helper' doesn't mean anything, lets nix it, as has been done in usages of this class
```suggestion
public class ScopedWorkCoordinator {
```

Reply:
Thanks.  Changed.  (to me), helper means that this isn't taking ownership of anything... however, the class morphed and is no longer just a helper that contains static methods.